### ISSUE TITLE
 Failed to load Tensorflow Model 1! [TypeError: right operand of 'in' is not an object] 

### ISSUE THREAD

**Issue by BLOCKMATERIAL** [mrousavy/react-native-fast-tflite#2]
Problem Description:
When attempting to load a TensorFlow 1 model, an error occurs with the message "Failed to load Tensorflow Model 1! [TypeError: right operand of 'in' is not an object]".

Expected Behavior:
Expected the TensorFlow 1 model to load successfully without any errors.

Reproduction Steps:

Run the application/script where an attempt to load the TensorFlow 1 model is made.
Observe the error message "Failed to load Tensorflow Model 1! [TypeError: right operand of 'in' is not an object]".
Actual Behavior:
Instead of successfully loading the model, an error occurs with the message "Failed to load Tensorflow Model 1! [TypeError: right operand of 'in' is not an object]".

Additional Information:
 

Device: iPhone 14 Pro Max
CoreML: disabled 
Model: from your demo (model_movenet_singlepose_lightning_tflite_int8_4)

```
  "dependencies": {
    "@shopify/react-native-skia": "^0.1.202",
    "react": "18.2.0",
    "react-native": "0.72.4",
    "react-native-fast-tflite": "^0.3.0",
    "react-native-vision-camera": "^3.0.0-rc.8",
    "react-native-worklets-core": "^0.2.0"
  },
```

**Comment by mrousavy** [comment]
Oh whoops, yea my fault. let me fix this

**Comment by mrousavy** [comment]
Fixed in 0.3.1! 🎉
